website/docs/utilities/disclosure/partials/code/how-to-use.md

```diff
@@ -26,4 +28,8 @@ To actually work, you need an interactive element that can trigger a custom even
 
```

Comment:
To me, the first example doesn't seem that helpful or necessary. I would just keep the 2nd example which includes a button.

Reply:
agreed. i'll remove it